src/joycontrolstick.cpp

```diff
@@ -1985,7 +1985,7 @@ bool JoyControlStick::hasSameButtonsMouseCurve()
 {
     bool result = true;
 
-    JoyButton::JoyMouseCurve initialCurve = JoyButton::LinearCurve;
+    JoyButton::JoyMouseCurve initialCurve;
```

Comment:
Situation similar to previous one. Just remove declaration here and put declaration later